proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/server/rest/DigitalObjectResource.java

```diff
@@ -1021,6 +1021,25 @@ public Response getRaw(
     }
 
     /**
+     * removes RAW datastream from selected pids
+     *
+     * @see PurgeFedoraObject
+     */
+    @DELETE
+    @Path(DigitalObjectResourceApi.RAW_PATH)
+    @Produces({MediaType.APPLICATION_JSON})
+    public SmartGwtResponse<DigitalObject> deleteRawDatastream(
+            @QueryParam(DigitalObjectResourceApi.DELETE_PID_PARAM) String pid
+    ) throws IOException, PurgeException {
+        RemoteStorage fedora = RemoteStorage.getInstance(appConfig);
```

Comment:
Tato implementace pouze odstraní RAW DS. Upload PDF přidavá také PREVIEW DS. Ten se nebude odstraňovat? Pokud se uploaduje stále stejný typ obsahu (PDF), tak to nejspíš může takto zůstat.